lua/lspconfig/server_configurations/clangd.lua

```diff
@@ -21,6 +21,34 @@ local function switch_source_header(bufnr)
   end
 end
 
+local function symbol_info(bufnr)
+  bufnr = util.validate_bufnr(bufnr)
```

Comment:
```suggestion
local function symbol_info(bufnr)
  local bufnr = vim.api.nvim_get_current_buf()
```
in usage you just pass 0 so no need receive a param